### PR TITLE
[13.0][IMP] ddmrp add used in MRP smart button

### PR DESCRIPTION
**Pull request by TDu**
Add a smart button at the top of the buffer form to open the list of BOM,
same than on the product form.

### CODE REVIEW

File: ddmrp/models/stock_buffer.py [hunk 1]
```diff
@@ -371,6 +371,15 @@ def action_view_mrp_productions(self):
         result["domain"] = [("id", "in", mrp_production_ids.ids)]
         return result
 
+    product_type = fields.Selection(related="product_id.type", readonly=True)
+    used_in_bom_count = fields.Integer(related="product_id.used_in_bom_count")
+
+    def action_used_in_bom(self):
```
Comment (sebalix) [inline ddmrp/models/stock_buffer.py:377]: For methods that doesn't update records data and returns an action, I use `button_` prefix (like the `button_validate` in `stock` module, used from the UI, while the validation itself is handled by `action_done`).

**Review by sebalix — APPROVED** [review]
LG, one minor comment about my personal taste, but it's personal :)

**Review by mmequignon — APPROVED** [review]

**Comment by mmequignon** [comment]
travis is :apple:

**Review by LoisRForgeFlow — APPROVED** [review]
Tested, LGTM

**Comment by LoisRForgeFlow** [comment]
@TDu can you forward port it to v14?

**Pull request merged**